### DISCUSSION TITLE
Modelleerprincipes - P1. Informatieobjecten zijn herbruikbaar

### DISCUSSION THREAD

**Discussion by pmaria** [Geonovum/disgeo-imsor#57] [General] [answered]
Modelleerprincipe [[P1]](https://geonovum.github.io/disgeo-imsor/modelleerprincipes/#p1-informatieobjecten-zijn-herbruikbaar) - Informatieobjecten zijn (her)bruikbaar in verschillende registraties

Discussie uit expertgroepsessie:
> - Lennart: zou dit niet ambitieuzer kunnen? We kunnen zeggen dat als we het over dezelfde concepten hebben, dat we het ook expliciet aangeven. 
> - Michel: zoals een uniform unique identifier. Dit zou echt iets toevoegen aan de samenhang, als we kunnen zeggen dat we het over hetzelfde ding hebben. [voorstel voor aanpassing van dit principe]: 'we gaan niet voor een unique id, maar we maken het wel mogelijk om relaties te leggen wanneer we het hebben over hetzelfde object (bv, door 'sameAs' relaties toe te staan)'. 
> - Paul: ik lees dat je gegevens los wil trekken van de dingen waar ze betrekking op hebben. Alleen dan begrijp ik niet meer goed wat een informatieobject is. Wat is het 'herbruikbare' hier? lijkt te duiden op een ID, niet het object zelf.
Lennart: Dit is een bottom-up formulering, ik denk dat top-down hier beter bij past (als ik over hetzelfde object in de werkelijkheid meer informatie vastleg willen we deze semantische samenhang op zo'n manier realiseren dat we weten dat het over hetzelfde object gaat).
> - [Voorstel voor aanpassing van principe]: de term 'herbruikbaar' veranderen - denk liever aan iets zoals 'linkbaar' of 'combineerbaar'. Ook iets over metadata opnemen in deze stelling, en de formulering herzien. Ook kijken naar een invulling van dit principe, waarbij we bv iets kunnen zeggen over het opnemen van 'sameAs'-achtige relaties.
> - Paul: Een onderwerp als inwinningsregels zou ook een relatie hiermee kunnen hebben.

**Comment by lvdbrink** [comment]
> - Lennart: zou dit niet ambitieuzer kunnen? We kunnen zeggen dat als we het over dezelfde concepten hebben, dat we het ook expliciet aangeven.
> - Michel: zoals een uniform unique identifier. Dit zou echt iets toevoegen aan de samenhang, als we kunnen zeggen dat we het over hetzelfde ding hebben. [voorstel voor aanpassing van dit principe]: 'we gaan niet voor een unique id, maar we maken het wel mogelijk om relaties te leggen wanneer we het hebben over hetzelfde object (bv, door 'sameAs' relaties toe te staan)'.

Hier voel ik wel iets voor (beide punten).

**Comment by pmaria** [comment, marked as answer]
Voorstel voor herformulering:

Informatieobjecten zijn in samenhang bruikbaar. Daarmee bedoelen we dan dat 2 of meer informatieobjecten (op basis van verschillende objecttypes) over hetzelfde object, in samenhang te gebruiken zijn

**Comment by lvdbrink** [comment]
P1 is vastgesteld in [expertgroep 8 juli](https://github.com/Geonovum/disgeo-imsor/blob/master/overleg/08-07-2021-aantekeningen.md).
